src/components/MarsApp.js

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Navbar from './shared/navbar';
 import HomePage from './pages/home';
-import ExamplePage from './pages/example';
+import PhotosPage from './pages/Photos';
+import WeatherPage from './pages/Weather';
```

Comment:
Admittedly the conventions for this codebase are a bit confusing, but we're going to have all folder names and standard js files (e.g. `index.js`) be in `camelCase` and React Components in `PascalCase` (e.g. `src/components/pages/photos/PhotosPage.js`)